docs/server/push-duty.rst

```diff
@@ -7,13 +7,22 @@ The pushing of the server rotates each week to another developer. Current rotati
 * kumar
 * magopian
 * muffinresearch
+* nolski
 * tofumatt
 
+Check out the `Add-ons calendar <https://www.google.com/calendar/embed?src=mozilla.com_lr5jsh38i6dmr72uu4d1nv7dcc%40group.calendar.google.com&ctz=America/Vancouver>`_ for a list of events.
```

Comment:
Do you want to leave the `ctz=America/Vancouver` part out? Not sure how it'll behave though, without a predefined timezone?